src/OpenTelemetry/Sdk.cs

```diff
@@ -47,6 +48,16 @@ static Sdk()
         /// </summary>
         public static bool SuppressInstrumentation => SuppressInstrumentationScope.IsSuppressed;
 
+        /// <summary>
+        /// Gets meter to create instruments for reporting metrics on Sdk internal state.
+        /// </summary>
+        internal static Meter SdkInternalStateMeter { get; } = new Meter("SdkInternalStateMeter");
```

Comment:
```suggestion
        internal static Meter SdkInternalStateMeter { get; } = new Meter("OpenTelemetry.Sdk");
```

Reply:
or just OpenTelemetry as that is the name of the package.

Reply:
`OpenTelemetry` looks very generic. `OpenTelemetry.Sdk` sounds better.

Reply:
I think we should be more explicit in the name to avoid any conflict. Maybe `OpenTelemetry.Sdk.Internal`